### DISCUSSION TITLE
Installing features of packages does not work any more

### DISCUSSION THREAD

**Discussion by roussec** [microsoft/vcpkg#32866] [Q&A] [answered]
I want to install a package feature, e.g. vcpkg install imgui[sdl2]. This does not work any more. The feature is listed when executing vcpkg search imgui. Is there a new way to install features in classic mode? 

Thank you.

**Comment by Neumann-A** [comment]
```
$ ./vcpkg.exe install imgui[sdl2]
Computing installation plan...
imgui has no feature named sdl2.
```

There is no `sdl2` feature in port `imgui`. The features are named `sdl2-binding` and/or `sdl2-renderer-binding`

**Comment by roussec** [comment]
Thank you. But this was just an example. Not the actual command. Using macOs:

cr@MacBookProCr vcpkg % vcpkg search imgui

daxa[utils-imgui]                         The ImGUI Daxa utility
imgui                    1.89.7           Bloat-free Immediate Mode Graphical User interface for C++ with minimal de...
imgui[allegro5-binding]                   Make available Allegro5 binding
imgui[docking-experimental]               Build with docking support
imgui[dx10-binding]                       Make available DirectX10 binding
imgui[dx11-binding]                       Make available DirectX11 binding
imgui[dx12-binding]                       Make available DirectX12 binding
imgui[dx9-binding]                        Make available DirectX9 binding
imgui[freetype]                           Build font atlases using FreeType instead of stb_truetype
imgui[glfw-binding]                       Make available GLFW binding
imgui[glut-binding]                       Make available Glut binding
imgui[libigl-imgui]                       Install the libigl-imgui headers
imgui[metal-binding]                      Make available Metal binding
imgui[opengl2-binding]                    Make available OpenGL (legacy) binding
imgui[opengl3-binding]                    Make available OpenGL3/ES/ES2 (modern) binding
imgui[osx-binding]                        Make available OSX binding
imgui[sdl2-binding]                       Make available SDL2 binding
imgui[sdl2-renderer-binding]              Make available SDL2 Renderer binding
imgui[vulkan-binding]                     Make available Vulkan binding
imgui[wchar32]                            Use WCHAR32 instead of WCHAR16
imgui[win32-binding]                      Make available Win32 binding
imgui-sfml               2.5#3            ImGui binding for use with SFML
imguizmo                 1.83#1           Immediate mode 3D gizmo for scene editing and other controls based on Dear...
implot                   0.15             Advanced 2D Plotting for Dear ImGui
libigl[imgui]                             Build with imgui
magnum-integration[imgui]                 ImGuiIntegration library
milerius-sfml-imgui      1.1#6            imgui dll for sfml usage
The result may be outdated. Run `git pull` to get the latest results.
If your port is not listed, please open an issue at and/or consider making a pull request.    -  https://github.com/Microsoft/vcpkg/issues

cr@MacBookProCr vcpkg % vcpkg install imgui[sdl2-binding]
zsh: no matches found: imgui[sdl2-binding]
cr@MacBookProCr vcpkg %

Installing imgui without a feature works as expected

**Reply by Neumann-A** [reply]
```
$ ./vcpkg.exe install imgui[sdl2-binding]
Computing installation plan...
The following packages will be built and installed:
    imgui[core,sdl2-binding]:x64-windows-release -> 1.89.7
  * sdl2[base,core]:x64-windows-release -> 2.26.5
```

**Reply by Neumann-A** [reply]
Do you have a `vcpkg.json` somewhere?

**Reply by roussec** [reply]
Meanwhile I was able to test it on Windows where it works as it should as you also see. But not on Mac. I do not have a vcpkg. json

**Comment by dg0yt** [comment, marked as answer]
> ~~~
> cr@MacBookProCr vcpkg % vcpkg install imgui[sdl2-binding]
> zsh: no matches found: imgui[sdl2-binding]

So the command never reached `vcpkg`... In `zsh`, you have to use quotes:
~~~
vcpkg install 'imgui[sdl2-binding]'

**Reply by roussec** [reply]
Thank you. I cannot recall ever using '' when installing optional packages. Thanks again!
